LDKNodeMonday/View/Home/BitcoinView.swift

```diff
@@ -324,6 +228,70 @@ struct BalanceHeader: View {
     }
 }
 
+struct TransactionButtons: View {
+    @ObservedObject var viewModel: BitcoinViewModel
+    @State private var isReceiveSheetPresented = false
+
+    var body: some View {
+        HStack(alignment: .center) {
+
+            // Send button
+            NavigationLink(value: NavigationDestination.address) {
+                Button {
+                    // Optional button action if needed
+                } label: {
+                    Text("Send")
+                }.buttonStyle(
+                    BitcoinFilled(
+                        width: 120,
+                        tintColor: .accent,
+                        isCapsule: true
+                    )
+                ).allowsHitTesting(false)  // Required to enable NavigationLink to work
```

Comment:
interesting, never ran into this modifier but makes sense from taking a look

Reply:
I'm hoping to address this when we do the send screen(s).
We need to use NavigationLink instead of a plain Button to use the NavigationAddress.
But I think we can simplify inside the Send views so this is not necessary.